crates/wasm/postBuild.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+WASM_BINARY=./target/wasm32-unknown-unknown/release/${pname}.wasm
```

Comment:
This won't work if you build in debug mode. Though, it might be that debug mode doesn't allow us to compile to wasm because of the "section too big" error?